pbp.sh

```diff
@@ -1,27 +1,14 @@
-echo "$(tput setaf 5)Pulling and bundling consumer-layouts...$(tput sgr0)"
-cd consumer-layouts
-git pull
-bundle install
+#!/usr/bin/env bash
 
-echo "$(tput setaf 5)Pulling and bundling consumer-domain...$(tput sgr0)"
-cd ../consumer-domain
-git pull
-bundle install
+REPOS=(consumer-layouts consumer-domain consumer-session domain-values consumer-main)
```

Comment:
Gotcha, I tested it locally and it seemed good, but zsh might've been interfering somehow.